recipes/numpy_quaddtype/meta.yaml

```
@@ -0,0 +1,64 @@
+{% set name = "numpy_quaddtype" %}
+{% set version = "0.0.4" %}
+{% set python_req = "python >=3.10,<3.13" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/SwayamInSync/numpy-user-dtypes/archive/refs/tags/quaddtype-v{{ version }}.tar.gz
+  sha256: a400f35cf980965486f9dd00c9d2289e877a4bd09a848f49b82de23e1855beea
+
+build:
+  number: 0
+  script: |
+    cd quaddtype
+    {{ PYTHON }} -m pip install . -vv --no-build-isolation
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ stdlib("c") }}
+    - python
+    - pip
+    - meson
+    - meson-python
+    - setuptools
+    - sleef ==3.6.1
+    - numpy >=2.0
+  host:
+    - {{ python_req }}
+    - pip
+    - setuptools
+    - meson
+    - meson-python
+    - sleef ==3.6.1
+    - numpy >=2.0
+  run:
+    - {{ python_req }}
+    - {{ pin_compatible('sleef') }}
+    - {{ pin_compatible('numpy') }}
```

If you are using numpy 2.0 you don't need this.
```suggestion
    - numpy
```